SmartEVSE-3/data/index.html

```diff
@@ -158,7 +168,7 @@ <h1 id="version" class="h5 mb-0 text-white-800">Version: </h1>
                                     <div class="col mr-2">
                                         <div class="text font-weight-bold text-primary text-uppercase mb-1">EVSE</div>
                                         <div class="row no-gutters align-items-center">
-                                          <div class="col-auto h5 mb-0 mr-3 font-weight-bold text-gray-800" style="width: 100px;">
+                                          <div class="col-auto h5 mb-0 mr-3 font-weight-bold text-gray-800" style="width: 120px;">
```

Comment:
In the past I changed some "width: 120px" to "width: 100px" because on my Samsung phone the layout was crippled (text wraps to new line because screen not big enough). 
Now I don't want to enforce "my" layout preferences, but I don't think it is wise to ping pong from 120 -> 100 -> 120 px in our git history. Should we go for a "responsive" html (so different layouts for phone/PC), or does that at too much complication / overhead in the code? Not my specialty to be honest ....

Don't let this comment block the merge, IMHO it is not a core problem...

Reply:
I initially set it to 120 as that was the width required to have everything aligned because of the "Connected:" title.
With 100px it looks strange as that value is indented differently than the others.
I can live with smaller width but than we need to rename "connected" to something smaller.

Reply:
connected
attached
coupled
hitched
linked

The shorter the synonyms, the worse they get to my taste :-)

Reply:
Exactly, so I sticked with connected and added a few pixels as my OCD does not allow bad alignments ;)